src/types/inp.ts

```diff
@@ -116,6 +150,26 @@ export interface INPAttribution {
    * (e.g. usually in the `dom-interactive` phase) it can result in long delays.
    */
   loadState: LoadState;
+  /**
+   * The total duration of Long Animation Frame scripts that intersect the INP
+   * duration. Note, this is limited to scripts > 5 milliseconds.
+   */
+  scriptsDuration?: number;
```

Comment:
I wonder if this should be called `totalScriptDuration` to make it clear that it could be a sum from more than one script entry.

Reply:
Done.